package.json

```diff
@@ -19,7 +19,8 @@
     "prepare": "husky install",
     "start:cypress": "tsc -p tsconfig.backend.json && NODE_ENV=production node --loader ts-node/esm --es-module-specifier-resolution=node node_dist/backend/cypress-server.js",
     "run:cypress": "cypress run",
-    "cypress": "npm-run-all -p start:cypress -r run:cypress"
+    "cypress": "npm-run-all -p start:cypress -r run:cypress",
+    "typecheck": "tsc --noEmit -p src/frontend/tsconfig.json"
```

Comment:
La til et script for å typesjekke, litt enklere å bruke når man sletter mye ting osv